### PR TITLE
Codespaces Optmizations

### PR DESCRIPTION
**Pull request by ericboehs**
- **Codespaces: Create on-create.sh**
- **Codespaces: Start Rails/Sidekiq on attach**
- **Codespaces: Prewarn bootsnap and add homebrew to zshrc**
- **Codespaces: open preview on rails server port open**
- **:retab file**
- **Codespaces: fix asdf -> zshrc syntax**
- **Codespaces: Missing packages; run apt update**
- **Codespaces: Use ruby image**
- **Codespaces: Remove asdf**
- **Codespaces: ruby 3.2**
- **Codespaces: change back to notify on port 3000 open**
- **Codespaces: open codespaces readme**
- **Codespaces: Implode RVM**
- **Codespaces: Move RVM implosion as it didn't exist yet**
- **Codespaces: RVM doesn't like to be sudoed**
- **Codespaces: remove rvm implosion; not working**
- **Codespaces: Remove unused docker extension**
- **Codespaces: Set ruby lsp version manager to none**

### CODE REVIEW

File: .devcontainer/post-create.sh [hunk 1]
```diff
@@ -1,51 +1,21 @@
...
+git -C /workspaces/vets-api-mockdata pull
+git -C /workspaces/vets-api pull
 
-sudo apt update
-sudo apt install -y libpq-dev pdftk shared-mime-info postgresql-15-postgis-3
+{
+{
+  "rubyLsp.rubyVersionManager": "none"
+}
+} >> /home/vscode/.vscode/settings.json
```
Comment (ericboehs) [inline .devcontainer/post-create.sh:16]: This isn't working as expected. Might need to put it in the repo dir instead and gitignore it.

File: .devcontainer/post-start.sh [hunk 2]
```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+# Ruby-lsp extension thinks we're using rbenv but we're not
+sudo rbenv implode --force
```
Comment (ericboehs) [inline .devcontainer/post-start.sh:4]: I can also remove this.

**Comment by stevenjcumming** [comment]
It appears that you can't run it in the browser and locally, so I think you should consider updating the docs with some troubleshooting instructions 

Screenshot 2024-03-28 at 10 44 54 AM

**Comment by stevenjcumming** [comment]
Also received this message in the browser

Screenshot 2024-03-28 at 10 31 18 AM

**Comment by ericboehs** [comment]
> It appears that you can't run it in the browser and locally, so I think you should consider updating the docs with some troubleshooting instructions
> 
> Screenshot 2024-03-28 at 10 44 54 AM

That's correct. I started to add something to do the docs but the scenario seemed a little arcane as I wrote out the solution.

**Review by stevenjcumming — APPROVED** [review]

**Pull request merged**